leveldb/external_test.go

```diff
@@ -10,8 +10,8 @@ import (
 	. "github.com/onsi/ginkgo"
```

Comment:
Note: ginkgo and gomega are test-only dependencies, not sure if we want to maintain internal copies of them as well.